scanpointgenerator/core/compoundgenerator.py

```diff
@@ -55,8 +55,8 @@ def __init__(self, generators, excluders, mutators):
         logging.debug("CompoundGenerator periods")
         logging.debug(self.periods)
 
-        self.position_units = generators[0].position_units.copy()
```

Comment:
I think this is a remnant from when it was NestedGenerator. It can actually be moved into the main for loop at line 34 now.